src/SprykerEco/Yves/Computop/Plugin/PayuCeeSingleSubFormPlugin.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Yves\Computop\Plugin;
+
+use Spryker\Yves\Kernel\AbstractPlugin;
+use Spryker\Yves\StepEngine\Dependency\Plugin\Form\SubFormPluginInterface;
+use SprykerEco\Yves\Computop\Form\DataProvider\PayuCeeSingleFormDataProvider;
+use SprykerEco\Yves\Computop\Form\PayuCeeSingleSubForm;
+
+/**
+ * @method \SprykerEco\Yves\Computop\ComputopFactory getFactory()
+ */
+class PayuCeeSingleSubFormPlugin extends AbstractPlugin implements SubFormPluginInterface
+{
+    /**
```

Comment:
* Add the specification to the methods and `{@inheritdoc}` document block.

Reply:
Done